source/appendices/a-glossary-of-terms.rst

```diff
@@ -459,7 +459,7 @@ Note: The main Code of Practice document takes precedence in the case of any con
      -
 
    * - Full_Content_Database
-     - A COUNTER Host_Type for report providers that offer databases that are a collection of content items that are not otherwise part of a serial or monograph (i.e. non-aggregated).
+     - A COUNTER Host_Type for report providers that offer databases that are a collection of content items that are not aggregated into titles (i.e. not part of a serial or book or other title).
```

Comment:
Is it intentional that the "Full_Content_Database may include multimedia materials" isn't included here?

Reply:
It's a source of confusion and led to questions about when full content database should be used in place of multimedia collection.